src/main/java/com/vaadin/training/bugrap/ui/BugrapApplicationModel.java

```diff
@@ -24,6 +29,8 @@
 
 public class BugrapApplicationModel {
```

Comment:
At some point in the future we might want to split this into smaller state models. each of them centered around certain component or functionality. This is something we might wanna keep in mind for the future reference, so no need to do anything regarding that now

Reply:
Ok, I had the same feeling.